app/src/main/java/de/dhbw/corona_world_app/ui/statistic/StatisticFragment.java

```diff
@@ -40,24 +44,18 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         statisticCallViewModel =
                 new ViewModelProvider(requireActivity()).get(StatisticCallViewModel.class);
 
-        //TODO remove this
-        if (!statisticCallViewModel.isInit()) {
+        if (statisticCallViewModel.isNotInit()) {
             try {
                 statisticCallViewModel.init(requireActivity().getFilesDir(), ThreadPoolHandler.getInstance());
             } catch (IOException e) {
-                e.printStackTrace();
+                ErrorDialog.createBasicErrorDialog(getContext(), "We were not able to create new Files", "Please restart the application or it will not function properly", null);
```

Comment:
Very nice error message but I think a "Please restart the app" without the second part suffices

Reply:
We should discuss if we make standardized error codes (e.g. "Error F212" or something along those lines)